server/src/services/app_import_export.service.ts

```diff
@@ -565,6 +566,31 @@ export class AppImportExportService {
     return appResourceMappings;
   }
 
+  // This is to migrate few properties from styles to properties as per new designs
```

Comment:
Lets provide clear documentation for the function, including its purpose, input parameters, and examples.
```suggestion
  /**
 * Migrates visibility, disabledState, and tooltip styles to properties based on component type.
 * @param {Component} component - The component object to process.
 * @param {string[]} componentTypes - An array of component types that should undergo migration.
 * @returns {Object} An object containing updated properties, styles, and general objects.
 */
```